scipy/stats/_morestats.py

```diff
@@ -4023,14 +4023,24 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate', *,
     return res
 
 
-def directionalmean(samples, *, axis=0, normalize=True):
+DirectionalStats = namedtuple('DirectionalStats',
+                              ('directional_mean', 'magnitude'))
```

Comment:
I think it's ok to call `directional_mean` just `mean` if `magnitude` is just `magnitude`. Although "magnitude" is somewhat ambiguous.

Reply:
I think it is a bit tricky to name. Maybe, we can change the name of the function to `directional_mean` and return a `MeanResultant` object with attributes `direction` and `magnitude` (as you suggested [here](https://github.com/scipy/scipy/pull/17126#discussion_r985128691))?

Reply:
I am a bit afraid that If we call it `magnitude` , typical users who do not read the docs carefully will interpret it as the length of the direction vector and then be confused that it is not 1. In literature, I always saw it coined as `mean_resultant_length`. That one will be foreign to most people but at least they would not assume something wrong. WDYT?

Reply:
↓

Reply:
> I am a bit afraid that If we call it `magnitude` , typical users who do not read the docs carefully will interpret it as the length of the mean direction vector and then be confused that it is not 1. In literature, I always saw it coined as `mean_resultant_length`. That one will be foreign to most people but at least they would not assume something wrong. WDYT?

Thanks for the suggestions @dschmitz89! `mean_resultant_length` sounds good. @mdhaber I can make the change (`mean` -> `mean_resultant` and `magnitude` -> `mean_resultant_length`) if it sounds good to you too.

> ↓

Hmm, I don't get it...

Reply:
I meant the down arrow to suggest "see comment below", but I didn't want to sound like a broken record.

Reply:
I changed `directional_mean` -> `mean_resultant` and `magnitude` -> `mean_resultant_magnitude`. Is it better now @mdhaber, @dschmitz89?

Reply:
> Thanks for the suggestions @dschmitz89! `mean_resultant_length` sounds good. @mdhaber I can make the change (`mean` -> `mean_resultant` and `magnitude` -> `mean_resultant_length`) if it sounds good to you too.

Sorry, missed this comment. I have changed `mean_resultant_magnitude` to `mean_resultant_length`.